samples/bluetooth/mesh/light/Kconfig

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Bluetooth mesh light sample"
+
+config BT_MESH_LIGHT_SAMPLE_NRF52_DFU
+	bool "Enable SMP DFU support"
+	depends on MCUMGR_CMD_OS_MGMT
+	depends on MCUMGR_CMD_IMG_MGMT
+	depends on MCUMGR_SMP_BT
+	depends on SOC_SERIES_NRF52X
```

Comment:
I've no strong opinion on this, but what if we don't enforce 52 series here. Then it should be possible to run this code with nRF53DK. I guess `smp_dfu.c` could be conditionally compiled in if, for example, `CONFIG_MCUMGR_SMP_BT` is enabled.

Reply:
I get your point. The issue is Thingy 53. I wanted to create a clear distinction between the dfu for thingy53 and the other platforms. Picking 52 series was the most easy option (only excluding `nrf5340dk_nrf5340`)

Reply:
I answered Ingar on this bellow ⬇️